src/main/java/org/sonarlint/intellij/BootstrapStartupActivity.java

```diff
@@ -33,9 +34,13 @@
 
   @Override
   public void runActivity(@NotNull Project project) {
+    if (ApplicationManager.getApplication().isUnitTestMode()) {
```

Comment:
Is it really necessary? Won't it remove a lot of coverage?

Reply:
It won't. All the services loaded by this class already have dedicated tests.

The problem is that this method was called automatically in light tests when the project is created, performing undesired side effects.